src/ci/docker/dist-x86_64-redox/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:16.04
```

Comment:
I think this is now equivalent to the `cross` container, right? If not, can you edit the scripts here to use the shared scripts in the `src/ci/docker/shared` folder?